docs/references/authorization.md

```diff
@@ -14,7 +14,9 @@ After the installation two users are available: `admin` and `epinio`, both with
 
 ## Switch user
 
-To switch users you need to set the `user` and `pass` keys of the Epinio settings file, located at `~/.config/epinio/settings.yaml`
+To switch users you need to set the `user` and `pass` keys of the Epinio settings file, located at `~/.config/epinio/settings.yaml`.
+The password should be base64 encoded. Below, `cGFzc3dvcmQ=` is the base64 encoded version of `password`.
```

Comment:
`should` means that a password not encoded in base64 is supported, i.e. the system auto-detects the format and acts accordingly. Looking at the code PR this however is not the case.
Therefore
```suggestion
The password has to be base64 encoded. Below, `cGFzc3dvcmQ=` is the base64 encoded version of `password`.
```